### PR TITLE
[OSCD] Powershell without powershell.exe Rule Added

### PR DESCRIPTION
**Pull request by nsaddler**
[OSCD Initiative] Develop Sigma rules for PowerShell Abuse #575: Task #75 (PowerShell without PowerShell.exe).

### CODE REVIEW

File: rules/windows/sysmon/sysmon_powershell_without_powershell_exe.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+title: PowerShell without PowerShell.exe
+id: 3f07b9d1-2082-4c56-9277-613a621983cc
+description: Detects loading Powershell packet by non-Powershell process
+references:
+    - https://speakerdeck.com/heirhabarov/hunting-for-powershell-abuse
+tag:
+    - attack.defense_evasion
```
Comment (yugoslavskiy) [inline rules/windows/sysmon/sysmon_powershell_without_powershell_exe.yml:7]: Hello @nsaddler! 

Thank you for your contribution! 
I believe that you can add a technique tag for PowerShell execution — `attack.t1059.001`

File: rules/windows/sysmon/sysmon_powershell_without_powershell_exe.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        - '\powershell.exe'
+        - '\powershell_ise.exe'
+        - '\sqlps.exe'
+        - '\sdiagnhost.exe'
+        - '\wsmprovhost.exe'
+        - '\winrshost.exe'
+        - '\mscorsvw.exe'
+        - '\syncappvpublishingserver.exe'
+        - '\runscripthelper.exe'
+        - '\ServerManager.exe'
```
Comment (yugoslavskiy) [inline rules/windows/sysmon/sysmon_powershell_without_powershell_exe.yml:31]: Could you please explain why did you decide to not use commandline filters, that are present in the original rule? thank you!

**Comment by e6e6e** [comment]
Hello Natalia!
This is a good rule, thank you! But it is very similar to [this rule](
https://github.com/Neo23x0/sigma/blob/de5444a81e770ec730aa5e3af69781ab222f021a/rules/windows/image_load/sysmon_in_memory_powershell.yml). I think it's better to change this PR (update the existing rule).

File: rules/windows/image_load/sysmon_in_memory_powershell.yml [hunk 3]
```diff
@@ -4,7 +4,7 @@ status: experimental
 description: Detects loading of essential DLL used by PowerShell, but not by the process powershell.exe. Detects meterpreter's "load powershell" extension.
 author: Tom Kern, oscd.community
```
Comment (yugoslavskiy) [inline rules/windows/image_load/sysmon_in_memory_powershell.yml:5]: Hello @nsaddler! I believe that you can put here your name as well as oscd.community  (:

Comment (nsaddler) [reply rules/windows/image_load/sysmon_in_memory_powershell.yml]: Thank you! Added

Comment (yugoslavskiy) [reply rules/windows/image_load/sysmon_in_memory_powershell.yml]: Thanks!

**Comment by yugoslavskiy** [comment]
Great job, @nsaddler! Thank you very much!

**Review by yugoslavskiy — APPROVED** [review]
@thomaspatzke @Neo23x0 guys, I believe that this PR could be merged (:
please let us know what do you think about it if there is anything you would like to change.

**Pull request merged**